core/lib/protobuf_config/src/da_client.rs

```diff
@@ -73,9 +76,17 @@ impl ProtoRepr for proto::DataAvailabilityClient {
                     .context("wait_for_finalization")?,
                 authenticated: *required(&conf.authenticated).context("authenticated")?,
                 verify_cert: *required(&conf.verify_cert).context("verify_cert")?,
-                path_to_points: required(&conf.path_to_points)
-                    .context("path_to_points")?
-                    .clone(),
+                points_source: match conf.points_source.clone() {
+                    Some(proto::eigen_config::PointsSource::Path(path)) => {
+                        let path = required(&path.path).context("path")?;
+                        zksync_config::configs::da_client::eigen::PointsSource::Path(path.clone())
+                    }
+                    Some(proto::eigen_config::PointsSource::Link(link)) => {
+                        let link = required(&link.link).context("link")?;
+                        zksync_config::configs::da_client::eigen::PointsSource::Link(link.clone())
+                    }
+                    None => return Err(anyhow::anyhow!("Invalid Eigen DA configuration")),
```

Comment:
why Anyhow instead of adding a proper error type?

Reply:
That is how it is done for the other clients